lib/errors.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = (translate, options) => {
+
+  var logger = (options || {}).logger;
+
+  if (typeof translate !== 'function') {
+    throw new Error('Errorhandler needs a translate function');
+  }
+
+  return (err, req, res) => {
+    var content = {};
+
+    if (err.code === 'SESSION_TIMEOUT') {
+      content.title = translate('errors.session.title');
+      content.message = translate('errors.session.message');
+    }
+
+    if (err.code === 'NO_COOKIES') {
+      err.status = 403;
+      content.title = translate('errors.cookies-required.title');
+      content.message = translate('errors.cookies-required.message');
+    }
+
+    content.title = content.title || translate('errors.default.title');
+    content.message = content.message || translate('errors.default.message');
```

Comment:
How useful is a default message? I wonder if in the future we should try to get something out out of the err object.

In lev-web we're currenltly doing something like this:
```js
    content.message = content.message || err.message || err || translate('errors.default.message');
```

Which isn't pretty but gives you some info when debugging.

I'm not sure what security would make of it.

Maybe it needs to be conditional on config.env?